src/test/java/seedu/address/model/ModelManagerTest.java

```diff
@@ -113,16 +112,18 @@ public void equals() {
         // different types -> returns false
         assertFalse(modelManager.equals(5));
 
-        // different addressBook -> returns false
-        // assertFalse(modelManager.equals(new ModelManager(differentModuleList, userPrefs)));
+        // different moduleList -> returns false
+        assertFalse(modelManager.equals(new ModelManager(differentModuleList, userPrefs)));
+
+        // Note : No NameContainsKeywordsPredicate class for Module yet.
```

Comment:
We should discuss what predicates to use for this class

Reply:
Okay, sure.